### ISSUE TITLE
Token didn't expire even after a day which is supposed to in one hour

### ISSUE THREAD

**Issue by Naveen95** [tymondesigns/jwt-auth#1837]
### Your environment
| Q                 | A
| ----------------- | ---
| Bug?              | yes
| New Feature?      | no 
| Framework         | Laravel
| Framework version | 5.8
| Package version   | 1.0.0
| PHP version       | 7.3.1

### Expected behaviour
The token must expire in one hour. 

### Actual behaviour
The token didn't expire even after a day. and still works fine. the cache driver is set to file.

**Comment by mfn** [comment]
How did you set the expiration? Did you set the `ttl` via `config/jwt.php`?

**Comment by Naveen95** [comment]
@mfn Yes i did via config/jwt.php. still it is not working.

**Comment by mfn** [comment]
When you paste the token to https://jwt.io/ (the debugger), what does it say? Is the expire correct? Please paste the outcome here

**Comment by Naveen95** [comment]
Yes i have set the time to expire as one min and the exp also denotes the same.
{
  "iss": "http://127.0.0.1:8000/api/auth/login",
  "iat": 1562217784,
  "exp": 1562217844,
  "nbf": 1562217784,
  "jti": "85stFvYA1Frh3NKl",
  "sub": 1,
  "prv": "87e0af1ef9fd15812fdec97153a14e0b047546aa"
}

But the token works even after a day. Not sure what is causing the problem

**Comment by ash-jc-allen** [comment]
@Naveen95 I'm not sure if you're still having any problems with this. But I used to get similar issues with tokens due to time drift on my virtual machine. Usually resyncing the VMs time worked. I don't know if this will be at all useful?
